src/event.rs

```diff
@@ -23,6 +26,43 @@ use std::fmt::{self, Debug, Formatter};
 // FIXME - See https://maidsafe.atlassian.net/browse/MAID-2026 for info on removing this exclusion.
 #[allow(clippy::large_enum_variant)]
 pub enum Event {
+    // =========== Client Events - Routing won't handle these ===========
+    /// Inform the user (library) that we are connected to a client
+    ConnectedToClient {
+        /// Client's endpoint
+        peer_addr: SocketAddr,
+    },
+    /// Inform the user (library) that we are disconnected from a client
+    ConnectionFailureToClient {
+        /// Client's endpoint
+        peer_addr: SocketAddr,
+    },
+    /// Inform the user (library) that we have a new message from a client
+    NewMessageFromClient {
+        /// Client's endpoint
+        peer_addr: SocketAddr,
+        /// Client's message
+        msg: NetworkBytes,
+    },
+    /// Inform the user (library) that we couldn't send this message to a client
+    UnsentUserMsgToClient {
```

Comment:
Thought would maintain symmetry with quic-p2p and routing types so that it's not named in different ways in different places. It's already `UnsentUserMsg` between routing and quic-p2p and hence this for the vaults.

Again, I'm OK to call it SendError if you want (i don't feel strongly either way)

Reply:
No worries let's keep that pattern for now and if needed we can change in all crates.